gov.pnnl.goss.gridappsd/src/gov/pnnl/goss/gridappsd/api/LogManager.java

```diff
@@ -44,7 +44,7 @@
 public interface LogManager {
 
 	
-	void log(LogMessage message);
+	void log(LogMessage message, String username);
```

Comment:
Provide inline documentation for what is expected from the implementation of this method and how the get and log method should be paired together.

Reply:
Done